packages/config/config/devices/0x021f/dmdp1.json

```diff
@@ -14,6 +14,28 @@
 		"max": "255.255"
 	},
 	"supportsZWavePlus": true,
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic_Set",
+			"description": "The Door/Window Sensor sends a Basic Set command to Association Group 2 (or the Control Group) to directly trigger devices (like a light, chime, etc.) in response to events.",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Notification_Report",
+			"description": "The Door/Window Sensor sends it a NOTIFICATION_REPORT when the door either opens or closes.",
```

Comment:
We try to avoid the uppercase notation for CC names. Prefer `Notification Report` over `NOTIFICATION_REPORT`.